frontend/src/assets/css/typography.scss

```diff
@@ -0,0 +1,82 @@
+@import url("https://fonts.googleapis.com/css2?family=Plus+Jakarta+Sans:wght@400;600;700;800&display=swap");
+
+:root {
+  --font-family: "Plus Jakarta Sans";
```

Comment:
All variables should be in the same place (vars.scss)

Reply:
Did I understand correctly that i need to completely remove `:root`. And to use variables, I will need to import vars in the desired file via `@import "vars"`?